TrueCraft.Client/TrueCraftGame.cs

```diff
@@ -79,16 +77,16 @@ protected override void Initialize()
             SpriteBatch = new SpriteBatch(GraphicsDevice);
             base.Initialize(); // (calls LoadContent)
             ChunkConverter = new ChunkRenderer(Graphics.GraphicsDevice, Client.World.World.BlockRepository);
-            Client.ChunkLoaded += (sender, e) => ChunkConverter.QueueChunk(e.Chunk);
-            Client.ChunkModified += (sender, e) => ChunkConverter.QueueHighPriorityChunk(e.Chunk);
-            ChunkConverter.MeshGenerated += ChunkConverter_MeshGenerated;
+            Client.ChunkLoaded += (sender, e) => ChunkConverter.Enqueue(e.Chunk);
+            Client.ChunkModified += (sender, e) => ChunkConverter.Enqueue(e.Chunk, true);
+            ChunkConverter.MeshCompleted += ChunkConverter_MeshGenerated;
             ChunkConverter.Start();
             Client.PropertyChanged += HandleClientPropertyChanged;
             Client.Connect(EndPoint);
             var centerX = GraphicsDevice.Viewport.Width / 2;
             var centerY = GraphicsDevice.Viewport.Height / 2;
             Mouse.SetPosition(centerX, centerY);
-            Camera = new Camera(GraphicsDevice.Viewport.AspectRatio, 70.0f, 0.1f, 1000.0f);
+            Camera = new Camera(GraphicsDevice.Viewport.AspectRatio, 70.0f, 1.0f, 1000.0f);
```

Comment:
Can someone tinker with Minecraft and try to guess what their near plane distance is?